watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/metab/metab_global_costing.yaml

```diff
@@ -0,0 +1,55 @@
+base_currency: USD_2020
+base_period: year
+defined_flows:
+  electricity:
+    value: 0.08
+    units: USD_2020/kWh
+  heat:
+    value: 0.0000061
+    units: USD_2020/kJ
+  H2_product:
+    value: -0.733
```

Comment:
I'd raise this to at least -2, matching the levelized cost of hydrogen for current conventional methods (steam methane reforming), but should actually be higher. I have some values to choose from if we want to replace here.